.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -11,13 +11,15 @@ body:
     attributes:
       value: |
         # Thank you for opening an issue.
-
+        
         The [OpenTofu](https://github.com/opentofu/opentofu) issue tracker is reserved for bug reports relating to the core OpenTofu CLI application and configuration language.
-
+        
         ## Filing a bug report
-
+        
+        **⚠ Warning!** Do not post `TRACE` level OpenTofu logs! They may contain sensitive credentials!
```

Comment:
We have this request later on:

> Full debug output can be obtained by running OpenTofu with the environment variable `TF_LOG=trace`. Please create a GitHub Gist containing the debug output. Please do _not_ paste the debug output in the issue, since debug output is long. Debug output may contain sensitive information. Please review it before posting publicly.

BTW we already have that warning